src/routers/blogAppV1.ts

```diff
@@ -0,0 +1,11 @@
+
+import express from 'express';
+import loginRouter from '../routes/loginRouter';
+import signUpRouter from '../routes/signUpRouter';
+
+const apiRouter = express.Router();
+
+apiRouter.use('/v1', loginRouter);
+apiRouter.use('/v1', signUpRouter);
```

Comment:
We can implement the versioning in the `app.ts` just to avoid adding n-times the versioning, but for now it is ok.